### PR TITLE
Offers

### PR DESCRIPTION
**Pull request by furilo**

### CODE REVIEW

File: _careers/backend-developer.md [hunk 1]
```diff
@@ -4,13 +4,16 @@ title: Ruby On Rails Developer
 date: 2019-03-18
 outtro: no
 active: true
+place: "Remote (Spain), onsite (Madrid), or mixed"
 ---
 
 ## About Populate
 
-We are a small design and tech consultancy building digital products. We do projects around digital strategy, UX, (open) data, visualization... We work together with organizations to define their digital presence and tell their stories with great UX, data visualization and fine code.
+We are a design and tech consultancy building digital products around civic engagement. We are hiring for [Gobierto Contracts](contratos.gobierto.es), which is our main focus right now: a tool to help public administrations control, plan and research about their contracting activity. Read [El por qué de Gobierto Contratación: hacer más eficiente la contratación pública](https://gobierto.es/blog/20210527-gobierto-contratacion.html) (spanish) about our motivation for building this tool — making public spending more efficient, nothing less!
```
Comment (ferblape) [inline _careers/backend-developer.md]:
```suggestion
We are a design and tech consultancy building digital products around civic engagement. We are hiring for [Gobierto Contracts](https://contratos.gobierto.es), which is our main focus right now: a tool to help public administrations control, plan and research about their contracting activity. Read [El por qué de Gobierto Contratación: hacer más eficiente la contratación pública](https://gobierto.es/blog/20210527-gobierto-contratacion.html) (spanish) about our motivation for building this tool — making public spending more efficient, nothing less!
```

File: _careers/data.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+date: 2021-07-01
+outtro: no
+active: true
+place: "Remote (Spain), onsite (Madrid), or mixed"
+---
+
+
+## About Populate
+
+We are a design and tech consultancy building digital products around civic engagement. We are hiring for [Gobierto Contracts](contratos.gobierto.es), which is our main focus right now: a tool to help public administrations control, plan and research about their contracting activity. Read [El por qué de Gobierto Contratación: hacer más eficiente la contratación pública](https://gobierto.es/blog/20210527-gobierto-contratacion.html) (spanish) about our motivation for building this tool — making public spending more efficient, nothing less!
```
Comment (ferblape) [inline _careers/data.md]:
```suggestion
We are a design and tech consultancy building digital products around civic engagement. We are hiring for [Gobierto Contracts](https://contratos.gobierto.es), which is our main focus right now: a tool to help public administrations control, plan and research about their contracting activity. Read [El por qué de Gobierto Contratación: hacer más eficiente la contratación pública](https://gobierto.es/blog/20210527-gobierto-contratacion.html) (spanish) about our motivation for building this tool — making public spending more efficient, nothing less!
```

File: _careers/data.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+We work on [Github](http://github.com/populatetools/) using a git workflow with CI/CD, tests, different automations... Our workflow has been polished throughout a few years, so have automated many things (monitoring, error tracking, server setups & updates, deployments, various checks...). You can take a look at the [Gobierto repo to see how we actually work](http://github.com/populatetools/gobierto).
+
+
+## Day to day work
+
+- **Plan** new features of Gobierto and existing applications, and contribute to new apps from scratch
+- **Implement** defined features, manage your sprints, do sprint reviews
+- **Collaborate** with other team members: provide feedback and do code reviews
+- **Improve** and maintain existing code of production applications
```
Comment (ferblape) [inline _careers/data.md]: These are developer tasks, we could change them to:

- Plan: analyze and suggest new analysis over the data
- Implement: develop your scripts and visualizations, prepare notebooks
- Present results: present results to the team, review suggestions and iterate

File: _careers/data.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+- Browse around in our general issue tracker (Github)
+- Get assigned initial issues to navigate around our development process: tests, deploy, code reviews...
+
+
+## Your first quarter
+
+- Master all of our internal tools and processes
+- You are tasked with planning a medium-sized feature in Gobierto that will take you several weeks to implement
+- Propose the work plan and work with a designer and a front-end engineer to implement it
+- Deploy code to staging, demo your work to the team/client, write documentation, polish it and merge it to master
```
Comment (ferblape) [inline _careers/data.md]: - You understand Gobierto contratos data model
- You are tasked with some small projects 
- A project has been released and is in production, adding value to the product

File: _careers/data.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+## Your first year
+
+- Responsible of one or more apps/modules/big features
+- Master Gobierto and have provided ideas to improve code and features
+- Speak with clients autonomously if needed
+- Do fluent code reviews
+- Have made suggestions to improve our processes
+- Have attended some conferences and proposed books to our book club
+- Given internal talks about tech and non-tech things
+- Have traveled with the rest of the team to one or more retreats (if global pandemics allow it)
```
Comment (ferblape) [inline _careers/data.md]: - Responsible of data in Populate
- Our workflows and pipelines are more mature
- Creating new data projects is a smooth process
- Data analysis has been used in different projects, not only Gobierto Contratos
- There are different ML models deployed

**Review by ferblape — CHANGES_REQUESTED** [review]
A ver que te parecen estas sugerencias

File: _careers/data.md (hunk 3 above)
Comment (furilo) [reply _careers/data.md]: Perfect

File: _careers/data.md (hunk 4 above)
Comment (furilo) [reply _careers/data.md]: ok

File: _careers/data.md (hunk 5 above)
Comment (furilo) [reply _careers/data.md]: just change the things :D

**Comment by ferblape** [comment]
Feedback pushed, ready to review again

**Pull request merged**